teams/developer-relations/guide-to-being-a-manager/README.md

```diff
@@ -90,49 +90,110 @@ With regards to this checklist, reviewers should do the following:
 * Ensure that stage labels are being properly marked
 * When entire checklist is checked off for an Epic, then that Epic should be completed and linked within a pull request to `master` to indicate completion
 
-### General Checklist
+### General Checklist \(Applies to All Stages\)
 
-#### General Items \(Apply to All Stages\)
+Before going into a stage-by-stage breakdown, here's a general overview of things to keep in mind.
 
-* [ ] High-schooler friendly writing style, easy to understand
-* [ ] Correct numbering
-* [ ] Every card has code\* 
-* [ ] 1 scroll per card\*\*
-* [ ] Proper grammar, punctuation, capitalization, etc.
-* [ ] Acceptable Styling per the Pull Request Checklist
+#### Writing Style
 
-\*Exception being hard and medium cards in labs as well as activity cards that just introduce concepts
+First and foremost, we want all our developers to use a conversational writing style when making cards. This is because the target audience for our curriculum is high school students, and our goal is to make technical education as accessible as possible. Writing as if you're having a conversation will keep students more engaged. When in doubt, think **blog post** instead of **calculus textbook**.
```

Comment:
Our audience is not high school students exactly, but actually underrepresented learners of all backgrounds. there should also be a mention of boring lectures as an example.